openpype/settings/entities/schemas/projects_schema/schema_project_max.json

```
@@ -65,6 +65,104 @@
+                    "label": "Review Height"
+                },
+                {
+                    "type": "number",
+                    "key": "percentSize",
+                    "label": "Percent of Output"
+                },
+                {
+                    "type": "boolean",
+                    "key": "keep_images",
+                    "label": "Keep Image Sequences"
+                },
+                {
+                  "key": "image_format",
+                  "label": "Image Format Options",
+                  "type": "enum",
+                  "multiselection": false,
+                  "defaults": "exr",
+                  "enum_items": [
+                    {"exr":  "exr"},
+                    {"jpg": "jpg"},
+                    {"png":  "png"},
+                    {"tga":  "tga"}
+                  ]
+                },
+                {
+                    "key": "visual_style",
+                    "label": "Preference",
+                    "type": "enum",
+                    "multiselection": false,
+                    "defaults": "Realistic",
+                    "enum_items": [
+                      {"Realistic":  "Realistic"},
+                      {"Shaded": "Shaded"},
+                      {"Facets":  "Facets"},
+                      {"ConsistentColors":  "ConsistentColors"},
+                      {"HiddenLine":  "HiddenLine"},
+                      {"Wireframe": "Wireframe"},
+                      {"BoundingBox": "BoundingBox"},
+                      {"Ink": "Ink"},
+                      {"ColorInk": "ColorInk"},
+                      {"Acrylic": "Acrylic"},
+                      {"Tech": "Tech"},
+                      {"Graphite": "Graphite"},
+                      {"ColorPencil": "ColorPencil"},
+                      {"Pastel": "Pastel"},
+                      {"Clay": "Clay"},
+                      {"ModelAssist": "ModelAssist"}
+                    ]
+                },
+                {
+                    "key": "viewport_preset",
+                    "label": "Pre-View Preset",
+                    "type": "enum",
+                    "multiselection": false,
+                    "defaults": "Quality",
+                    "enum_items": [
+                      {"Quality":  "Quality"},
+                      {"Standard": "Standard"},
+                      {"Performance":  "Performance"},
+                      {"DXMode":  "DXMode"},
+                      {"Customize":  "Customize"}
+                    ]
+                },
+                {
+                    "key": "anti_aliasing",
+                    "label": "Anti-aliasing Quality",
+                    "type": "enum",
+                    "multiselection": false,
+                    "defaults": "None",
+                    "enum_items": [
+                      {"None":  "None"},
+                      {"2X": "2X"},
+                      {"4X":  "4X"},
+                      {"8X":  "8X"}
```

There's a lot of double spaces here?
```suggestion
                      {"Quality": "Quality"},
                      {"Standard": "Standard"},
                      {"Performance": "Performance"},
                      {"DXMode": "DXMode"},
                      {"Customize": "Customize"}
                    ]
                },
                {
                    "key": "anti_aliasing",
                    "label": "Anti-aliasing Quality",
                    "type": "enum",
                    "multiselection": false,
                    "defaults": "None",
                    "enum_items": [
                      {"None": "None"},
                      {"2X": "2X"},
                      {"4X": "4X"},
                      {"8X": "8X"}
```

---

I didn't realize that until you point out.....Some careless mistake by me